examples/parse_folder.rs

```diff
@@ -0,0 +1,70 @@
+/// This an example usage of the rustpython_parser crate.
+/// This program crawls over a directory of python files and
+/// tries to parse them into an abstract syntax tree (AST)
+///
+/// example usage:
+/// $ RUST_LOG=info cargo run --release parse_folder /usr/lib/python3.7
+
+#[macro_use]
+extern crate clap;
+extern crate env_logger;
+#[macro_use]
+extern crate log;
+
+use clap::{App, Arg};
+
+use rustpython_parser::{ast, parser};
+use std::path::Path;
+
+fn main() {
+    env_logger::init();
+    let app = App::new("RustPython")
+        .version(crate_version!())
+        .author(crate_authors!())
+        .about("Walks over all .py files in a folder, and parses them.")
+        .arg(
+            Arg::with_name("folder")
+                .help("Folder to scan")
+                .required(true),
+        );
+    let matches = app.get_matches();
+
+    let folder = Path::new(matches.value_of("folder").unwrap());
+    if folder.exists() && folder.is_dir() {
+        println!("Parsing folder of python code: {:?}", folder);
+        let res = parse_folder(&folder).unwrap();
+        println!("Processed {:?} files", res.len());
```

Comment:
Should it print the parsed file or something?

Reply:
I have some ideas for metrics:
- parsed lines per second
- print the amount of functions / classes

For now, I just use it to try and scan my cpython lib folder